curiouslearning/assessment-survey-js
# Added Initial unit testing commit's using jest!

# Changes
- I have Added Initial unit testing commit's using jest!

# How to test
- User need to install the dependencies in their machines first using **npm install** Command,
- In order to test the changes user can run the **npm run test** command and in the terminal can see the test cases running!

Ref: [AJ-210](https://curiouslearning.atlassian.net/browse/AJ-210)


[AJ-210]: https://curiouslearning.atlassian.net/browse/AJ-210?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
